db/migrations/20220615035112-update-github-server-apps.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const tableName = "GitHubServerApps";
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.addColumn(tableName, "gitHubAppName", {
+			type: Sequelize.STRING,
+			allowNull: false
+		});
+
+		await queryInterface.addColumn(tableName, "installationId", {
+			type: Sequelize.INTEGER,
+			allowNull: false
+		});
+
+		await queryInterface.renameColumn(tableName, 'githubBaseUrl', 'gitHubBaseUrl');
+		await queryInterface.renameColumn(tableName, 'githubClientId', 'gitHubClientId');
```

Comment:
Adding columns is fine, but renaming columns might affect current production.  I don't think anyone is using the model though, so you might be safe?  Try it in Staging first.  If it doesn't work, the better way is to create the new column, copy the data over from the old to the new one, then after deploying that change, do another DB migration to remove the old column.